lib/engine/game/g_1882.rb

```diff
@@ -204,6 +204,10 @@ def action_processed(action)
         @corporations.reject! { |c| c.id == 'SC' }
         @sc_company = nil
       end
+
+      def neutral_tokens(corporation)
```

Comment:
Sorry to keep changing this on you but ventusignis suggestion of a custom method in the game class seemed a good one.

Probably worth having a token_note method that allows a string to be added for 1882 and in future other games.